zoe-core/src/functions/admin.kt

```diff
@@ -32,12 +33,29 @@ val listTopics = zoeFunction<AdminConfig, ListTopicsResponse>(name = "topics") {
     admin(config.props).use { cli ->
         cli.listTopics().listings().get()
             .map { it.name() }
-            .let(cli::describeTopics).all().get()
-            .map { (_, topic) ->
-                TopicDescription(
-                    topic.name(),
-                    topic.isInternal,
-                    topic.partitions().map { it.partition() })
+            .let { topics ->
+                val describeTopicsFuture = cli
+                    .describeTopics(topics).all()
+
+                val describeConfigsFuture = cli
+                    .describeConfigs(topics.map { ConfigResource(ConfigResource.Type.TOPIC, it) }).all()
+
+                describeTopicsFuture.get() to describeConfigsFuture.get()
+            }.let { (kafkaTopicDescriptions, configResources) ->
```

Comment:
cosmetic detail: I feel it would be better to merge the two `.let` blocks into one. What do you think?
```suggestion
            val (kafkaTopicDescriptions, configResources) = describeTopicsFuture.get() to describeConfigsFuture.get()
```